lighthouse-core/lib/url-shim.js

```diff
@@ -215,6 +215,18 @@ class URLShim extends URL {
     const urlScheme = protocol.includes(':') ? protocol.slice(0, protocol.indexOf(':')) : protocol;
     return NON_NETWORK_SCHEMES.includes(urlScheme);
   }
+
+  /**
+   * @param {string} src
+   * @return {boolean}
+   */
+  static isSvgUrl(src) {
```

Comment:
you're too fast :)

> I was thinking we still put this on mimeType until v9 and extract /\.(png|jpeg|jpg|svg|webp|gif|avif)(\?.*)?$/.match(url)[1] so no usage changes are required (and data equivalent)

WDYT? ironically this is sometimes even more accurate than the `record.mimeType` because AVIF and WebP are often sent as `application/octet-stream` by older servers 😆

Reply:
Checking `image.mimeType === 'image/svg+xml'` was the only use case for `mimeType` in our audits. I hesitant to implement a mime type guesser for every extension if we only need svg. Are you still concerned that plugins will need *something* to reference for `mimeTypes`?

Reply:
> Are you still concerned that plugins will need something to reference for mimeTypes?

That was indeed my concern, and it seemed simple enough to avoid the breakage. I also think we might use it to enable the pixel-based estimates for the audits based on `optimized-images` gatherer in snapshot mode.